docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +12,49 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-1]]
+=== Logstash 6.1.1 Release Notes
+*  There are no user-facing changes in Logstash core in this release.
+
+[float]
+==== Input Plugins
+
+*`Beats`*:
+
+* Fix issue with close_wait connections, making sure keep alive sent back to the client. (https://github.com/logstash-plugins/logstash-input-beats/pull/272[#272])
+
+*`HTTP`*:
+
+* In the event that all webserver threads are busy this plugin will now return a 429, busy, error. (https://github.com/logstash-plugins/logstash-input-http/pull/75[#75])
+
+*`JDBC`*:
+
+* Fix connection and memory leak. (https://github.com/logstash-plugins/logstash-input-jdbc/issues/251[#251])
+
+*`Syslog`*:
+
+* Fix issue where stopping a pipeline. (e.g., while reloading configuration) with active inbound syslog connections could cause Logstash to crash. (https://github.com/logstash-plugins/logstash-input-syslog/issues/40[#40])
+
```

Comment:
There's a stray period after pipeline. I'd also shift the example :

"Fix issue where stopping a pipeline with active inbound syslog connections (for example, while reloading the configuration) could cause Logstash to crash.